schemas/stsci.edu/asdf/transform/label_mapper-1.3.0.yaml

```diff
@@ -0,0 +1,149 @@
+%YAML 1.1
```

Comment:
We don't need to add this file anymore, right?

Reply:
I wasn't sure if these still required a new version since they are linked in an updated example so I kept them, but I'll remove them now and see if the tests pass